Lib/test/test_memoryview.py

```diff
@@ -534,11 +534,6 @@ def test_memoryview_hex(self):
         m2 = m1[::-1]
         self.assertEqual(m2.hex(), '30' * 200000)
 
-    def test_copy(self):
```

Comment:
https://github.com/python/cpython/commit/f81be8aa3ff3cbae66686482219bd7c7561f3de8

Even if `property` object is allowed for copy operation, `property` object does not allow to be pickled.

Reply:
```
>>> import pickle
>>> import copy
>>> a = property()
>>> pickle.dumps(a)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: cannot pickle 'property' object
>>> copy.copy(a)
<property object at 0x101017230>
>>> copy.deepcopy(a)
<property object at 0x101017230>
```